lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -2,7 +2,13 @@
 <div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden" tabindex="-1">
     <div class="msg">
         <div class="msg-content">
-            <h3 class="title"><%- gettext("Your team could not be created!") %></h3>
+            <h3 class="title">
+                <% if (isNew) { %>
+                    <%- gettext("Your team could not be created!") %>
```

Comment:
Please replace the exclamation marks in these strings with periods. (Line 7 and line 9)
"Your team could not be created." "Your team could not be updated."